app/frontend/components/common/contactForm.js

```diff
@@ -55,6 +59,13 @@ class ContactForm extends Component {
             method: 'POST',
             target: '_blank',
             action: `//formspree.io/${ADALITE_CONFIG.ADALITE_SUPPORT_EMAIL}`,
+            onSubmit: () => {
+              this.setState({sumbitted: true})
```

Comment:
Maybe you could create a method which changes the state of the the contactForm. It's just a detail but we dont use setState method inside render method anywhere in the project, so again just to stay consistent :).

Reply:
This is the code from the issue branch that this branch depends on so it's not mine, can you put this comment there so the author decides? :)
-> https://github.com/vacuumlabs/adalite/pull/497